dev/benchmarks/macrobenchmarks/lib/src/backdrop_filter.dart

```diff
@@ -0,0 +1,109 @@
+import 'dart:ui';
+
+import 'package:flutter/material.dart';
+
+class BackdropFilterPage extends StatefulWidget {
+  @override
+  _BackdropFilterPageState createState() => _BackdropFilterPageState();
+}
+
+class _BackdropFilterPageState extends State<BackdropFilterPage> with TickerProviderStateMixin {
+  static const int numRows = 15;
+  static const int numCols = 5;
+  bool _blurGroup = false;
+  bool _blurTexts = true;
+  AnimationController animation;
+
+  @override
+  void initState() {
+    super.initState();
+    animation = AnimationController(vsync: this, duration: const Duration(seconds: 1));
+    animation.repeat();
+  }
+
+  @override
+  void dispose() {
+    animation.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    Widget addBlur(Widget child, bool shouldBlur) {
+      if (shouldBlur) {
+        return ClipRect(
+          child: BackdropFilter(
+            filter: ImageFilter.blur(sigmaX: 5, sigmaY: 5),
+            child: child,
+          ),
+        );
+      } else {
+        return child;
+      }
+    }
+
+    Widget text() {
+      return addBlur(Column(
+        children: <Widget>[
+          Container(padding: const EdgeInsets.all(5), child: const Text('txt')),
+        ],
+      ), _blurTexts);
+    }
+
+    Widget col() {
+      return Column(children: List<Widget>.generate(numRows, (int i) => text()));
+    }
+
+    return Scaffold(
+      backgroundColor: Colors.grey,
```

Comment:
Can we add some background that could tell the difference between with and without a `BackdropFilter`? For example, put 10000 '0's in the stack below the `BackdropFilter` as https://api.flutter.dev/flutter/widgets/BackdropFilter-class.html did.

Otherwise looks good.

Reply:
I added the background text (in yellow) and rand some numbers on my Moto E2 and did another commit.